src/azul/logging.py

```diff
@@ -6,9 +6,45 @@
 )
 
 
+class LambdaLoggerFilter(logging.Filter):
+
+    def __init__(self, app: AzulChaliceApp = None):
+        try:
+            # From https://github.com/aws/aws-lambda-base-images/tree/python3.8
+            # Imports global variable present in lambda environment
+            from bootstrap import (  # noqa:AZUL141
+                _GLOBAL_AWS_REQUEST_ID,
+            )
+        except ImportError:
+            # Block exists for local development/lack of aws_request_id variable
+            self.aws_request_id = "local"
+        else:
+            assert app.lambda_context.aws_request_id == _GLOBAL_AWS_REQUEST_ID
+            self.aws_request_id = _GLOBAL_AWS_REQUEST_ID
```

Comment:
@baladithyab @hannes-ucsc for parking lot;

If we already have the app.lambda_context.aws_request_id when this code runs in a deployment, why import from bootstrap.py?

Reply:
@baladithyab after speaking with Hannes about this during parking lot, we agreed to remove the import of the protected global variable, use app.lambda_context.aws_request_id instead.